app/src/main/java/com/solux/flory/presentation/gift/send/SelectBouquetActivity.kt

```diff
@@ -2,16 +2,29 @@ package com.solux.flory.presentation.gift.send
 
 import android.os.Bundle
 import androidx.activity.viewModels
-import com.solux.flory.databinding.ActivityGiftSelectbouquetBinding
+import coil.load
+import coil.transform.CircleCropTransformation
+import com.solux.flory.databinding.ActivityGiftSelectBouquetBinding
+import com.solux.flory.presentation.profile.NeighborInfo
 import com.solux.flory.util.base.BindingActivity
 
-class SelectBouquetActivity : BindingActivity<ActivityGiftSelectbouquetBinding>(ActivityGiftSelectbouquetBinding::inflate){
+class SelectBouquetActivity : BindingActivity<ActivityGiftSelectBouquetBinding>(ActivityGiftSelectBouquetBinding::inflate){
     private val viewModel by viewModels<BouquetViewModel>()
     private lateinit var adapter: BouquetAdapter
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+
         initAdapter()
         impossibleBtnClick()
+        
+        val NEIGHBOR_KEY = "selectedNeighbor"
```

Comment:
상수는 하단에 이렇게 작성해주시는겁니다!!
```suggestion
        companion object {
            const val NEIGHBOR_KEY = "selectedNeighbor"
        }
```